manifests/capture.augen.yaml

```
@@ -0,0 +1,14 @@
+name: Capture Augen
+authors: Nova.
+homepage: https://github.com/Nova1422/plugins/tree/main?tab=readme-ov-file#captureaugen
+license: ALL-RIGHTS-RESERVED
+version: v1.0.0-capture.augen
+shortDescription: Lets you capture the Super Heavy gegno ship "Augen" without consequences.
+description: Lets you capture the Super Heavy gegno ship Augen without consequences.
+url: https://github.com/Nova1422/plugins/releases/download/v1.0.0-capture.augen/capture.augen.zip
+iconUrl: https://raw.githubusercontent.com/Nova1422/plugins/v1.0.0-capture.augen/myplugins/capture.augen/icon.png
+autoupdate:
+  type: tag
```

```suggestion
  type: tag
  update_url: https://github.com/Nova1422/plugins
```
If `homepage` doesn't point to the repo, you need to specify `update_url` (same for the other manifests)

---

Not to tell you how to do your work but, #1846. It had similar format, auto update seemed to have worked for it